firebase-firestore/src/main/java/com/google/firebase/firestore/bundle/BundleReader.java

```diff
@@ -153,29 +152,50 @@ private int indexOfOpenBracket() {
     }
   }
 
+  private class ReadJsonResult {
```

Comment:
drive-by comment: this class should be `static`, and probably `final`.

Reply:
Done.